plugins/events-backend/report.api.md

```diff
@@ -18,7 +18,7 @@ import { LoggerService } from '@backstage/backend-plugin-api';
 // @public @deprecated
 export class DefaultEventBroker implements EventBroker {
   // @deprecated
-  constructor(logger: LoggerService, events?: EventsService);
+  constructor(logger: LoggerService, config?: Config, events?: EventsService);
```

Comment:
Can't break the API like this. Let's leave the config out of this one and instead hardcode it to `'never'` for the old event broker implementation.